karma.conf.js

```diff
@@ -37,6 +37,11 @@ module.exports = config => {
           .ignore('./lib/esm-utils.js')
           .ignore('path')
           .ignore('supports-color')
+          .ignore('./lib/buffered-runner.js')
+          .ignore('./lib/reporters/buffered.js')
+          .ignore('./lib/serializer.js')
+          .ignore('./lib/worker.js')
+          .ignore('./lib/pool.js')
```

Comment:
`workerpool` looks browser compatible enough to support our browser range. Is there a way to actually get these files in the browser bundle and get parallel runs in there as well?

I'm not super happy with keeping this blacklist for browser builds. It would be better if we could make a split where they are actually separate dependency graphs to make it less magic. I've already started doing some of that on the rollup branch, so I might just follow up on that thread there or later after both of those features are merged

Reply:
> workerpool looks browser compatible enough to support our browser range. Is there a way to actually get these files in the browser bundle and get parallel runs in there as well?

yes, it might work.  I didn't want to try it in this PR though.